### PR TITLE
Fix issue: rsyslog rate limit does not work on version 8.2110.0

### PR DESCRIPTION
**Pull request by Junchao-Mellanox**
#### Why I did it

The default stable version of rsyslog on bullseye has a bug about rate limit. It causes rate limit not work. The bug has been fixed on backport version 8.2206.0-1~bpo11+1.

Buster has no such issue.

#### How I did it

Upgrade rsyslog from 8.2110.0 to 8.2206.0-1~bpo11+1

#### How to verify it

Manual test

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [x] 202205

### CODE REVIEW

**Review by keboliu — APPROVED** [review]

File: build_debian.sh [hunk 1]
```diff
@@ -356,6 +355,10 @@ sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y in
     jq                      \
     auditd
 
+# default rsyslog version is 8.2110.0 which has a bug on log rate limit,
+# use backport version 8.2206.0-1~bpo11+1
+sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install rsyslog=8.2206.0-1~bpo11+1
```
Comment (qiluo-msft) [inline build_debian.sh:360]:
```suggestion
sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -t bullseye-backports -y install rsyslog
```

Comment (Junchao-Mellanox) [reply build_debian.sh]: Fixed

**Review by qiluo-msft — APPROVED** [review]

**Comment by Junchao-Mellanox** [comment]
Hi @qiluo-msft , could you please help merge this? All checkers are passed.

**Comment by liushilongbuaa** [comment]
@Junchao-Mellanox , bullseye-backports is removed by debian.
branch <= 202411 are blocked.

**Pull request merged**